server/src/test/java/org/apache/druid/server/coordinator/rules/BroadcastDistributionRuleSerdeTest.java

```diff
@@ -44,15 +43,15 @@
   public static List<Object[]> constructorFeeder()
   {
     return Lists.newArrayList(
-        new Object[]{new ForeverBroadcastDistributionRule(ImmutableList.of("large_source1", "large_source2"))},
-        new Object[]{new ForeverBroadcastDistributionRule(ImmutableList.of())},
-        new Object[]{new ForeverBroadcastDistributionRule(null)},
-        new Object[]{new IntervalBroadcastDistributionRule(Intervals.of("0/1000"), ImmutableList.of("large_source"))},
-        new Object[]{new IntervalBroadcastDistributionRule(Intervals.of("0/1000"), ImmutableList.of())},
-        new Object[]{new IntervalBroadcastDistributionRule(Intervals.of("0/1000"), null)},
-        new Object[]{new PeriodBroadcastDistributionRule(new Period(1000), null, ImmutableList.of("large_source"))},
-        new Object[]{new PeriodBroadcastDistributionRule(new Period(1000), null, ImmutableList.of())},
-        new Object[]{new PeriodBroadcastDistributionRule(new Period(1000), null, null)}
+        new Object[]{new ForeverBroadcastDistributionRule()},
+        new Object[]{new ForeverBroadcastDistributionRule()},
+        new Object[]{new ForeverBroadcastDistributionRule()},
```

Comment:
These seem duplicate. Could you clean up these parameters? Same for the other below param sets.

Reply:
Cleaned up params